src/script/lua_api/l_object.cpp

```diff
@@ -1086,32 +1105,20 @@ int ObjectRef::l_get_player_name(lua_State *L)
 int ObjectRef::l_get_player_velocity(lua_State *L)
 {
 	NO_MAP_LOCK_REQUIRED;
-	ObjectRef *ref = checkobject(L, 1);
-	RemotePlayer *player = getplayer(ref);
-	if (player == NULL) {
-		lua_pushnil(L);
-		return 1;
-	}
-	// Do it
-	push_v3f(L, player->getSpeed() / BS);
-	return 1;
+
+	log_deprecated(L, "Deprecated call to get_player_velocity, use get_velocity instead.");
+
+	return l_get_velocity(L);
```

Comment:
Warnings will not be spammed continuously if you do the deprecation using aliases:
```C++
luaL_Reg ObjectRef::methods[] = {
...
	// New function first, deprecated function(s) after
	luamethod_aliased(ObjectRef, get_velocity, getvelocity),
	{"get_player_velocity", ObjectRef::l_get_velocity},
	luamethod_aliased(ObjectRef, set_acceleration, setacceleration),
}
```
Unfortunately there's yet no `define` that takes three function names (new + 2x old).